puppet/modules/web/templates/web.conf.erb

```diff
@@ -5,7 +5,6 @@
 
   DocumentRoot /var/www/vhosts/web/htdocs
   RewriteEngine on
-  RewriteRule   ^/projects(.*)  http://projects.theforeman.org/projects$1  [R,L]
```

Comment:
I'm not sure we can remove this redirect, sorry.  I just checked the web server's access log and see we're still getting a lot of requests for these URLs by users from search engines.  I think the reason is that we're only serving a temporary, not permanent redirect.

(`grep "GET /projects" /var/log/httpd/web_access.log`)